mobile/src/tasks/senderTask.ts

```
@@ -0,0 +1,20 @@
+    await new Promise(async (resolve) => {
+        while (BackgroundService.isRunning()) {
+            if (!dataQueue.isEmpty()) {
+                const dataFrame = dataQueue.dequeue();
+                if (dataFrame) {
+                    webSocketService.sendDataFrames([dataFrame]);
+                }
+            }
+            await sleep(100);
+        }
+        resolve(null);
+    });
```

Biome (2.1.2)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)